docs/docs/dev_docs/getting_started/aztecnr-getting-started.md

```diff
@@ -84,19 +54,14 @@ authors = [""]
 compiler_version = "0.16.0"
```

Comment:
this is wrong

Reply:
i just removed the compiler version field so it should always work